### PR TITLE
Update TextAnalytics to enable live testing in sovereign clouds for multiple services

### PR DESCRIPTION
**Pull request by v-hongli1**
These changes enable TextAnalytics to run live tests against Public, UsGov and China.

1. Update files : TextAnalyticsAsyncClientTest.java and TextAnalyticsClientTest.java
Fixed the `expected: <U***ER_CANCELLED> but: <***UCCE***FULLY_COMPLETED>` exception, so change the way of get `LongRunningOperationStatus` for the test case `TextAnalyticsAsyncClientTest.cancelHealthcareLro` and `TextAnalyticsClientTest.cancelHealthcareLro`.

2. Update file : test-resources.json
Match with the `ArmTemplateParameter` setting, so change the parameter name of the endpoint suffix to `cognitiveServicesEndpointSuffix`.

Pipeline results:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1165804&view=results

@benbp, @scottaddie, @joshfree, @meeraharidasa, @mayurid and @mssfang for notification.

### CODE REVIEW

**Comment by v-xuto** [comment]
@benbp Could you please review this PR?

File: sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java [hunk 1]
```diff
@@ -2259,8 +2259,12 @@ public void cancelHealthcareLro(HttpClient httpClient, TextAnalyticsServiceVersi
                 syncPoller = client.beginAnalyzeHealthcareEntities(documents, options).getSyncPoller();
             syncPoller = setPollInterval(syncPoller);
             syncPoller.cancelOperation();
+            LongRunningOperationStatus operationStatus = syncPoller.poll().getStatus();
+            while (!LongRunningOperationStatus.USER_CANCELLED.equals(operationStatus)) {
```
Comment (benbp) [inline sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java:2263]: I'm not familiar with the java tests, but I assume we would want some sort of test timeout here, should the `getStatus()` call fail to be cancelled.

**Comment by benbp** [comment]
@v-xuto I added one comment (more of a question), but the engsys related changes look good. You have a sign-off from the java team as well, so if you don't have permissions to merge let me know and I can do that.

**Comment by v-xuto** [comment]
@benbp I don't have permissions to merge this PR.

**Pull request merged**